apps/web-client/src/routes/+layout.ts

```diff
@@ -19,11 +19,14 @@ export const load: LayoutLoad = async ({ url }) => {
 	// If in browser, we init the db, otherwise this is a prerender, for which we're only building basic html skeleton
 	if (browser) {
 		// If COUCHDB_HOST port is defined, we can assume the "remote" CouchDB is running and we should connect to it.
-		const remoteDb = COUCHDB_HOST ? `http://${COUCHDB_USER}:${COUCHDB_PASSWORD}@${COUCHDB_HOST}:${COUCHDB_PORT}/${DB_NAME}` : undefined;
 
-		return {
-			db: await createDB().init({ name: "[DB_INIT]", debug: false }, { remoteDb })
-		};
+		const db = await createDB();
+		db.init({ name: "[DB_INIT]", debug: false });
+		db.replicate({ name: "[DB_INIT]", debug: false }).live(
+			`http://${COUCHDB_USER}:${COUCHDB_PASSWORD}@${COUCHDB_HOST}:${COUCHDB_PORT}/${DB_NAME}`
+		);
```

Comment:
Maybe it would be better to run the live replication, in `+layout.svelte`, after the initial replication is finished.